pytorch_lightning/metrics/functional/sequence.py

```diff
@@ -0,0 +1,29 @@
+from collections import Counter
+from typing import List
+
+import torch
+from torchtext.data.metrics import bleu_score as bleu
```

Comment:
this import shall be optional, pls check how it is done with sklearn metrics or loggers

Reply:
@Borda torchtext had changed to optional import